modules/shared/components/molecules/PostViewHeader/UserInfo/PostViewHeaderTypes.d.ts

```diff
@@ -0,0 +1,10 @@
+import type { IUserInfo } from '../../UserInfo/IUserInfo';
+
+declare namespace PostViewHeaderTypes {
+  export interface IProps extends IUserInfo.IProps {
```

Comment:
why didn't you add the rest of props here?

Reply:
it already takes the props from the IUserInfo

Reply:
I am from mobile and I didn't see the extends 😂